Meta/run.sh

```diff
@@ -340,7 +340,7 @@ $SERENITY_EXTRA_QEMU_ARGS
 -device isa-vga
 -device isa-ide
 $SERENITY_BOOT_DRIVE
--device i8042 
```

Comment:
Could you revert these whitespace changes? If there's some formatting cleanup to do in Meta/ it should be done as a separate commit or PR.